components/hyper_serde/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "hyper_serde"
+version = "0.13.2"
+edition = "2018"
+authors = ["Anthony Ramine <n.oxyde@gmail.com>"]
+description = "Serde support for hyper types."
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/servo/hyper_serde"
```

Comment:
This should probably change right?

Reply:
Oh yep. This is fixed in the latest version of the branch.